src/Kernel/Kernel.hh

```diff
@@ -86,13 +75,13 @@ public:
   // All Kernels are required to define the "kernelValue", "gradValue",
   // and "grad2Value" methods, with the same call signatures 
   // as these functions.
-  double kernelValue(double etaMagnitude, double Hdet) const;
-  double gradValue(double etaMagnitude, double Hdet) const;
-  double grad2Value(double etaMagnitude, double Hdet) const;
+  double kernelValue(double etaMagnitude, const double Hdet) const;
```

Comment:
Is there some reason why Hdet is const and not etaMagnitude? Is etaMagnitude changed in some of the routines?

Reply:
Offhand I think no particular reason.  We could make the etaMagnitude constant as well.  Is that a change you'd like to see?

Reply:
It doesn't matter that it happens now, but it would be nice to keep things consistent between TableKernel, SphericalTableKernel, and Kernel.